src/docfx/config/ops/OpsConfigLoader.cs

```diff
@@ -12,15 +12,16 @@ namespace Microsoft.Docs.Build
 {
     internal static class OpsConfigLoader
     {
-        public static OpsConfig? LoadOpsConfig(ErrorBuilder errors, Package package, string workingDirectory = ".")
+        public static OpsConfig? LoadOpsConfig(
+            ErrorBuilder errors, Package package, string workingDirectory = ".")
```

Comment:
Consider remove the `="."` default value since any code not passing `workingDirectory` would implicitly depend on the current working directory which is a potential bug when if we build different working directories in parallel in the future.

```suggestion
            ErrorBuilder errors, Package package, string workingDirectory)
```

Reply:
Done.